src/promptflow-devkit/promptflow/batch/_batch_engine.py

```diff
@@ -49,6 +49,7 @@
 from promptflow.contracts.flow import Flow
 from promptflow.contracts.run_info import FlowRunInfo, Status
 from promptflow.contracts.types import AttrDict
+from promptflow.evals._constants import PF_BATCH_TIMEOUT_SEC
```

Comment:
`promptflow-devkit` should not depend on `promptflow-evals`. Please revert the changes in `_batch_engine.py`.

Reply:
Will do.